bigchaindb/tendermint/lib.py

```diff
@@ -14,38 +14,76 @@
     from sha3 import sha3_256
 
 import requests
+
 import bigchaindb
 from bigchaindb import backend, config_utils
-from bigchaindb import Bigchain
 from bigchaindb.models import Transaction
 from bigchaindb.common.exceptions import (SchemaValidationError,
                                           ValidationError,
                                           DoubleSpend)
 from bigchaindb.tendermint.utils import encode_transaction, merkleroot
 from bigchaindb.tendermint import fastquery
 from bigchaindb import exceptions as core_exceptions
+from bigchaindb.consensus import BaseConsensusRules
 
 logger = logging.getLogger(__name__)
 
 
-class BigchainDB(Bigchain):
+class BigchainDB(object):
+    """Bigchain API
```

Comment:
It's called Bigchain (instead of BigchainDB) in some places because the project began its life as "Bigchain". The "DB" was added a couple of months after the start of the project (i.e. over two years ago). It would be okay to change "Bigchain" to "BigchainDB".